examples/blockly-vue3/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "blockly-vue3",
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
```

Comment:
Could you change this and the readme to `npm run start`? That's what we use in other examples =)

Reply:
Better? :smile:

Reply:
Oh yeah sorry, my bad, I totally forgot to update the README! Fixed!